Sources/PackageGraph/PackageGraph.swift

```diff
@@ -24,10 +24,10 @@ public struct PackageGraph {
 
     // FIXME: These are temporary.
     public let modules: [Module]
-    public let externalModules: Set<Module>
+    public let externalModules: [Module]
```

Comment:
This doesn't seem right, we don't need to have array semantics here because we already have all the modules in modules property.